client/user/src/mealorder/menu/meal-order-menu-container.html

```diff
@@ -0,0 +1,218 @@
+
+            paper-tab.iron-selected .day {
+                color: var(--paper-light-blue-500);
+            }
+
+            paper-tab .paper-tab-container {
+                @apply --layout-horizontal;
+                @apply --layout-center-center;
+                text-decoration: none;
+                color: #323232;
+                font-size: 1.5em;
+            }
+
+            paper-tab[disabled] .paper-tab-container {
+                color: var(--paper-grey-500);
+            }
+
+            .day-of-the-week {
+                font-size: 0.7em;
+                font-weight: lighter;
+            }
+
+            @media only screen and (max-width: 600px) {
+                paper-tabs {
+                    margin-top: 3%;
+                    width: 100%;
+                }
+            }
+        </style>
+
+        <paper-tabs id="paper-tabs" selected="[[date]]"
+                    attr-for-selected='data-key'>
+            <template is="dom-repeat" items="{{currentWeek}}" as="calendar">
+                <paper-tab class="paper-tab"
+                           data-key$="[[_getDate(calendar)]]"
+                           link disabled="[[!calendar.hasMenu]]">
+                    <a href="#/menu/[[_getDate(calendar)]]"
+                       class="paper-tab-container">
+                        <div>
+                            <div class="day-of-the-week">[[_getDayOfTheWeek(calendar)]]
+                            </div>
+                            <div class="day">[[_getDay(calendar)]]</div>
+                        </div>
+                    </a>
+                </paper-tab>
+            </template>
+        </paper-tabs>
+        <!--meal-order-vendor-menus-->
+    </template>
+    <script>
+        /**
+         * menu-order is container for the vendors' menus and navigation.
+         *
+         * Contains a menu for the current date and tabs for quick date navigation.
+         *
+         * @customElement
+         * @polymer
+         * @appliesMixin MealOrderApp.ReduxMixin
+         */
+        class MealOrderMenuContainer extends (Polymer.Element) {
+            static get is() {
+                return 'meal-order-menu-container';
+            }
+
+            static get properties() {
+                return {
+                    /**
+                     * The current server time for the calendar.
+                     */
+                    serverTime: {
+                        type: Object,
+                    },
+
+                    /**
+                     * Selected day.
+                     */
+                    date: {
+                        type: Object,
+                    },
+
+                    /**
+                     * Array of dates with menu which may not exist.
+                     *
+                     * @typedef {Object} Date
+                     * @typedef {Boolean} hasMenu If menu for this date exists
+                     */
+                    currentWeek: {
```

Comment:
I guess we don't use week, but the following 8 days. So it's just `displayedDays`